### DISCUSSION TITLE
Text selection in front-end error modal

### DISCUSSION THREAD

**Discussion by shennan** [gatsbyjs/gatsby#30099] [Ideas / Feature Requests] [answered]
Would be nice to be able to select text in the Gatsby front-end errors that pop up in the browser for quick Google/Issue searches. I'm referring to the modal errors once the page has been rendered.

My guess is that this is a simple change in the `user-select` property in the style sheets.

**Comment by LekoArts** [comment, marked as answer]
I've added it as part of the darkmode PR: https://github.com/gatsbyjs/gatsby/pull/30025/commits/47a72c65e90277be98d0c475557a97c71336a4c3

**Reply by shennan** [reply]
Thanks.
